lib/session.js

```diff
@@ -293,7 +295,10 @@ Session.prototype.save = function(fn) {
         if(session.rawSocket && session.store.socketIndex)
           session.store.socketIndex[sid] = session.rawSocket ;
         if (res.uid) {
-          userSessionIndex[res.uid] = session;
+          if (userSessionIndex[res.uid] === undefined) {
+            userSessionIndex[res.uid] = [];
```

Comment:
Shouldn't this be  `userSessionIndex[res.uid] = {};` (aka object, not array?)

Also, the line above should use the more standard way of checking for undefined `typeof .... === "undefined"`